javascript/big-practice/src/scripts/models/listNoteModel.js

```diff
@@ -0,0 +1,52 @@
+import NoteModel from './noteModel';
+
+/**
+ * @class listNoteModel
+ * @description manage data of note list
+ */
+export default class listNoteModel {
+  constructor(noteModel) {
+    this.notes = [
+      {
+        id: 0,
```

Comment:
Should define a file to contain data. We don't add data in this file because data will be returned from API, not in model